pkg/plugin/system/system.go

```diff
@@ -30,10 +30,11 @@ import (
 // PluginSystem is the interface that defines all the methods for the aspect CLI
 // plugin system intended to be used by the Core.
 type PluginSystem interface {
-	Configure(streams ioutils.Streams) error
+	ConfigurePluginSystem(streams ioutils.Streams) error
 	TearDown()
 	BESBackendInterceptor() interceptors.Interceptor
 	BuildHooksInterceptor(streams ioutils.Streams) interceptors.Interceptor
+	ConfigurePlugins() *aspecterrors.ErrorList
```

Comment:
Do we need 2 methods? Do we need to return all errors instead of just the first one?

Reply:
I had the same comment on the first review. Every time I look at this, I'm more convinced that a single method is what we want. I'd say that `Configure` should do both things and return a list of errors. I don't see why not return all errors possible.

Reply:
Done